docs/README.md

```diff
@@ -1,42 +1,55 @@
+# Loki Documentation
+
 <p align="center"> <img src="logo_and_name.png" alt="Loki Logo"> <br>
   <small>Like Prometheus, but for logs!</small> </p>
 
-Grafana Loki is a set of components, that can be composed into a fully featured
+Grafana Loki is a set of components that can be compsed into a fully featured
 logging stack.
 
-It builds around the idea of treating a single log line as-is. This means that
-instead of full-text indexing them, related logs are grouped using the same
-labels as in Prometheus. This is much more efficient and scales better.
-
-## Components
-- **[Loki](loki/README.md)**: The main server component is called Loki. It is
-  responsible for permanently storing the logs it is being shipped and it
-  executes the LogQL
-  queries from clients.
-  Loki shares its high-level architecture with Cortex, a highly scalable
-  Prometheus backend.
-- **[Promtail](promtail/README.md)**: To ship logs to a central place, an
-  agent is required. Promtail
-  is deployed to every node that should be monitored and sends the logs to Loki.
-  It also does important task of pre-processing the log lines, including
-  attaching labels to them for easier querying.
-- *Grafana*: The *Explore* feature of Grafana 6.0+ is the primary place of
-  contact between a human and Loki. It is used for discovering and analyzing
-  logs.
+Unlike other logging systems, Loki is built around the idea of only indexing
+labels for logs and leaving the original log message unindexed. This means
+that Loki is cheaper to operate and can be orders of magnitude more efficent.
```

Comment:
this particular statement is probably going to be hard to get where everyone likes it, though I might suggest something more like
> Unlike other logging systems, Loki is built around the idea of only indexing metadata about your logs: labels (just like Prometheus labels).  Log data itself is then compressed and stored in chunks in object stores such as S3 or GCS, or even locally on the filesystem.  A small index and highly compressed chunks simplifies the operation and significantly lowers the cost of Loki.